grommet/grommet
# Improved a11y for Layer, Select, TextArea and TextInput

#### What does this PR do?
Changes Layer Select TextArea and TextInput to have a better accessibility.

Today pressing esc inside a Select closes the Layer. So I decided to take a pass in this feature for all components.

#### Where should the reviewer start?
LayerContainer

#### What testing has been done on this PR?
manual

#### How should this be manually tested?
open layer stories and select stories and make sure you can use the keyboard consistently

#### Any background context you want to provide?

#### What are the relevant issues?
no issue, sorry

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
no

#### Should this PR be mentioned in the release notes?
yes

#### Is this change backwards compatible or is it a breaking change?
backwards
